leaflet-markercluster/index.d.ts

```diff
@@ -118,10 +116,7 @@ declare namespace L {
       * If you have customized the clusters icon to use some data from the contained markers,
       * and later that data changes, use this method to force a refresh of the cluster icons.
       */
-      refreshClusters():MarkerClusterGroup;
-      refreshClusters(layerGroup:LayerGroup<ILayer>):MarkerClusterGroup;
-      refreshClusters(marker: Marker):MarkerClusterGroup;
-      refreshClusters(markers: Marker[]):MarkerClusterGroup;
+      refreshClusters(clusters?: Marker | Marker[] | LayerGroup): this;
```

Comment:
This can take a `{[index: any]: L.Layer}` as well (I think that syntax is right).
Should fit `markers.refreshClusters({id_0: myMarker0, id_any: myMarker33});`